tests/test_besttrack.py

```diff
@@ -41,7 +41,7 @@ def test_from_atcf(mocker):
         atcf=input_directory / 'florence2018_atcf.trk', nws=8,
     )
 
-    assert best_track.storm_id == 'BT02008'
+    assert best_track.storm_id is None
```

Comment:
new storm ID system does not find `BT02008` from `WRT00001` in the storm table

Reply:
Why is that?